### PR TITLE
Fix so jQuery is Alphabetized correctly in technologies drop down

### LINKED ISSUE #2629: jQuery needs to be placed alphabetically in the Technologies filter on the Projects page
### Overview
As a developer, we should make sure the website is intuitive to use. In this issue, we will fix jQuery being placed last on the Technologies filter on the Projects page.

<details>

<summary>Current Technologies filter</summary>

technologies-filter-jQuery

</details>

### Action Items
- [ ] Fix the Technologies filter to display jQuery and the rest of the technologies in alphabetical order

### Resources/Instructions
- [Projects page](https://www.hackforla.org/projects/)
- [pages/projects-page.html](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/pages/projects-page.html) 
- [assets/js/current-projects.js](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/assets/js/current-projects.js)

### PR DESCRIPTION
**Pull request by sydneywalcoff**
Fixes #2629

### What changes did you make and why did you make them ?
  - created a lowercase sorted technologies and languages list to get the correct position of jQuery (I guess the array prototype sort method is based on the first capital letter it finds? so jQuery was sorted as 'Q')
  - removed incorrectly placed jQuery using splice
  - spliced 'jQuery' into the correct place in the original filterValues array

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screen Shot 2022-01-09 at 5 03 38 PM

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Screen Shot 2022-01-09 at 5 00 29 PM

</details>

### CODE REVIEW

**Comment by sydneywalcoff** [comment]
I noticed that a branch I reviewed made it into this somehow? How should I fix?

File: assets/js/current-projects.js [hunk 1]
```diff
@@ -19,13 +19,19 @@ document.addEventListener("DOMContentLoaded",function(){
...
-        for(const [filterName,filterValue] of Object.entries(filters)){
+        for(let [filterName,filterValue] of Object.entries(filters)){
             // Add displayed filter title, resolves issue of "program areas" not being valid html attribute name due to spacing
             let filterTitle = "";
             if(filterName === "programs"){
                 filterTitle = "program areas"
             } else if(filterName === 'technologies') {
                 filterTitle = 'languages / technologies'
+                const lowercaseValues = filterValue.map(value => value.toLowerCase())
+                lowercaseValues.sort()
```
Comment (Aveline-art) [inline assets/js/current-projects.js]: @sydneywalcoff I'm unsure if you have tried it yet, but `sort()` takes an optional [comparison function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description) as an argument to customize sort order.  If it works, it should future-proof further additions to the `filterValue` object, and prevent this from happening again in the long run.

Comment (sydneywalcoff) [reply assets/js/current-projects.js]: @Aveline-art Thanks! I didn't even think to adventure down that rabbit hole, but that's a *much* cleaner solution.

**Comment by SAUMILDHANKAR** [comment]
Review ETA: 1/14/22
Availability: Thursday 6-9 pm

File: _sass/components/_wins-page.scss [hunk 2]
```diff
@@ -490,6 +491,7 @@
             text-align: left;
             position: relative;
             left: 0;
+            bottom: 0px;
```
Comment (SAUMILDHANKAR) [inline _sass/components/_wins-page.scss]: Hi @sydneywalcoff, thanks for working on this issue. The changes made in _wins-page.scss file shouldn't be part of this PR. Please remove this commit.

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]
Great work on sorting jQuery in technology dropdown, however we need to remove the commit with changes to the wins page.

**Review by lopezpedres — COMMENTED** [review]
@sydneywalcoff ! LGTM. I'll just wait until the changes request from @SAUMILDHANKAR are done.  Great job!

File: _sass/components/_wins-page.scss (hunk 2 above)
Comment (sydneywalcoff) [reply _sass/components/_wins-page.scss]: @SAUMILDHANKAR what would be the easiest way to do this? Revert? Rebase?

**Comment by Aveline-art** [comment]
@sydneywalcoff Please resolve [merge conflicts ](https://github.com/hackforla/website/pull/2682/conflicts)that had resulted from your force push. If you need a guide, [this ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github)should get you started. 😃

File: assets/js/current-projects.js [hunk 3]
```diff
@@ -19,11 +19,22 @@ document.addEventListener("DOMContentLoaded",function(){
...
+                filterTitle = 'languages / technologies'
+                filterValue.sort((a,b)=> {
+                    a = a.toLowerCase()
+                    b = b.toLowerCase()
+                    if(a < b) return -1;
+                    if(a > b) return 1;
+                    return 0;
+                })
+            } else {
+                filterTitle = "program areas"
```
Comment (SAUMILDHANKAR) [inline assets/js/current-projects.js]: @sydneywalcoff Good job in removing the unwanted commit as suggested earlier. 
In line 37 of your code, filterTitle shouldn't be set to "program areas" as it has already been done above.

File: assets/js/current-projects.js [hunk 4]
```diff
@@ -19,11 +19,22 @@ document.addEventListener("DOMContentLoaded",function(){
...
             } else if(filterName === 'technologies') {
                 filterTitle = 'languages / technologies'
```
Comment (SAUMILDHANKAR) [inline assets/js/current-projects.js]: this else if looks like a repeat of line 27, 28

**Review by SAUMILDHANKAR — CHANGES_REQUESTED** [review]

**Comment by sydneywalcoff** [comment]
@SAUMILDHANKAR git is a _lot_ of fun haha. Should be good now

**Review by SAUMILDHANKAR — APPROVED** [review]

**Comment by SAUMILDHANKAR** [comment]
@sydneywalcoff Thanks for working on this issue and making all the changes. Looks good in my environment.

**Pull request merged**